src/linkplay/utils.py

```diff
@@ -74,6 +126,15 @@ def decode_hexstr(hexstr: str) -> str:
         return hexstr
 
 
+def decode_properties(dictionary: dict[Any, str], keys: List[Any]) -> dict[Any, str]:
```

Comment:
Added this as a convenience method, though I'm not sure if it'll be needed elsewhere.  Happy to remove if you'd prefer being more verbose in `bridge.py`

Reply:
Would you be able to give a bit of an example?  Sounds like  agreat idea, but my python is a bit rusty.

Reply:
Oh, wait.  I think I understand.  You're mainly talking about removing the `Any` I've brought in here.

Not a problem, I'll sort that out now

Reply:
I had the following in mind

```python
T = TypeVar("T")


def get_typed_dict(dictionary: dict[str, str], keys: T) -> dict[T, str]:
    """Returns a dictionary with the given keys typed to T."""
    d: dict[T, str] = {}
    for key in keys:
        d[key] = dictionary[key] if key in dictionary else ""

    return d
    
...

class LinkPlayDevice

    async def update_status(self) -> None:
        """Update the device status."""
        d: dict[str, str] = await self.bridge.json_request(
            LinkPlayCommand.DEVICE_STATUS
        )  # type: ignore[assignment]

        self.properties = get_typed_dict(d, DeviceAttribute)
```

This would ensure that the properties dict in the player is always filled with all keys in `DeviceAttribute`, defaulting to an empty string if not found, skipping over any unknown `DeviceAttribute`.

Reply:
Decoding the artist etc would follow after loading in the dict since it doesn't count for all properties